docs/content/misc/version-support-status.ngdoc

```diff
@@ -7,7 +7,11 @@
 This page describes the support status of the significant versions of AngularJS.
 
 <div class="alert alert-info">
-  On July 1, 2018 AngularJS entered a 3 year Long Term Support period.
+  On July 1, 2018 AngularJS entered a 3 year Long Term Support period.<br />
+  <br />
+  _**UPDATE (2020-07-27):**_<br />
```

Comment:
Should there be a set date e.g Dec 31, 2021 for the ultimate LTS as opposed to opaque end of 2021.
(like how originally the date was prices, i.e. June 30, 2021)

Reply:
Good idea - will update.